atlp-rwanda/bravo-bn-fe
# #182201677 Feature:reset-password

##  What does this PR do?
- Reset a password
##  Description of Task to be completed?
- design an interface will be used by a user to reset a password
## How should this be manually tested?
- Clone the repo.
- checkout to a branch called `ft-reset-password-182201677`.
- Run npm install to install modules.
- Run `npm start` to run an app.
- Find `locahost:8080/reset` and put your email to reset
-  Check your email then click on the link then put your password.

## pivot Tracker link
https://www.pivotaltracker.com/story/show/182201677
## Questions
- N/A.
